views/tasks/view.php

```diff
@@ -27,27 +37,34 @@
         <h4 class="head-regular">Отклики на задание</h4>
     <?php if ($task->responses): ?>
         <?php foreach ($task->responses as $response): ?>
-            <div class="response-card">
-                <img class="customer-photo" src="<?= $response->user->avatar; ?>" width="146" height="156" alt="Фото заказчиков">
-                <div class="feedback-wrapper">
-                    <a href="<?= Url::to(['/user/view/', 'id' => $response->user->id]); ?>" class="link link--block link--big"><?= $response->user->name; ?></a>
-                    <div class="response-wrapper">
-                        <div class="stars-rating small"><?= StarWidget::getStars(count($response->user->reviews)); ?></div><br>
-                        <p class="reviews"><?= count($response->user->reviews); ?> отзыва</p>
+            <?php if (Yii::$app->user->id === $task->customer_id ||  Yii::$app->user->id === $response->user_id): ?>
```

Comment:
когда появляется больше, чем одна проверка одного и того же, то целесообразно использовать `in_array`, это упростит код:
```suggestion
            <?php if (in_array(Yii::$app->user->id, [$task->customer_id, $response->user_id])): ?>
```